apps/example/lib/main.dart

```diff
@@ -1,6 +1,6 @@
 // import "package:example/firebase_options.dart";
 // import "package:firebase_core/firebase_core.dart";
-import "package:firebase_version_repository/firebase_version_repository.dart";
+// import "package:firebase_version_repository/firebase_version_repository.dart";
```

Comment:
I prefer if these are not commented out if they are not used

Reply:
They were used by mike or Niels I think. Shall I just remove all references to firebase or do you want me to spend time to make it work through a dotenv optionally?